RELEASES.md

```diff
@@ -58,26 +58,36 @@ A release should not be finalized until the development team and the external co
 
 The beginning of a new major release series is marked by the release of a new major version. A major release series is comprised of all minor and patch releases made under the same major version number. The series continues to receive bug fixes (released as minor or patch releases) until it reaches end of life. The date when a major release series reaches end of life is determined by one of the two following methods:
 
-- If the next major release is made within the first 6 months, then the end of life date of the major release series is 1 year after its initial release.
-- If the next major release is made 6 months after the initial release, then the end of life date of the major release series is 6 months after the release date of the next major release.
+- If the next major release is made within the first 6 months, then the end of life date of the major release series is 18 months year after its initial release.
+- If the next major release is made 6 months after the initial release, then the end of life date of the major release series is 12 months after the release date of the next major release.
 
 For example, if the current major release series is v1 and was released on January 1st, 2022, then v1 will be supported at least until January 1st, 2023. If v2 is published on August 1st 2022, then v1's end of life will be March 1st, 2023.
 
 Only the following major release series have a stable release status. All missing minor release versions have been discontinued.
 
+We reserve the right to drop support for releases if they are deemed unused (for example, because the Cosmos SDK version they depend on is not used or has been deprecated).
+
 ### ibc-go 
 
 |Release|End of Life Date|
 |-------|----------------|
-|`v7.2.x`|March 17, 2024|
-|`v7.3.x`|March 17, 2024|
-|`v8.0.x`|November 10, 2024|
+|`v7.2.x`|September 17, 2024|
+|`v7.3.x`|September 17, 2024|
+|`v8.0.x`|May 10, 2025|
 
 ### Callbacks middleware
 
 |Release|End of Life Date|
 |-------|----------------|
-|`v0.1.x-ibc-go-v7.3.x`|March 17, 2024|
+|`v0.1.x+ibc-go-v7.3.x`|September 17, 2024|
+|`v0.1.x+ibc-go-v8.0.x`|May 10, 2025|
+
+### `08-wasm` light client proxy module
+
+|Release|End of Life Date|
+|-------|----------------|
+|`v0.1.0+ibc-go-v7.3.x-wasmvm-v1.5.x`|September 17, 2024|
+|`v0.2.x+ibc-go-v8.0.x-wasmvm-v1.5.x`|May 10, 2025|
```

Comment:
should we add a note that we may drop pre v1.0 versions if we deem them unnecessary to maintain? Pre 1.0 versions are basically minor or major releases and really we are only looking to give long term support for major releases

Reply:
do we even want to explicitly talk about long term support? I feel like pre 1.0.0 releases are meant for rapid and potentially disruptive changes and long term support sounds antithetical to that and could potentially stifle it. Happy if a comment is at least added though, but wanted to raise this Q.

Reply:
I added a note [here](https://github.com/cosmos/ibc-go/pull/5606/commits/30a7cb17d166c8648896ef90a1bd18e554216895).